### PR TITLE
local image for avatar

### PR DESCRIPTION
**Pull request by adamdavies1915**

### CODE REVIEW

**Review by Jonzy3000 — CHANGES_REQUESTED** [review]
Hello @adamdavies1915, thank you for the pull request. I think having a nullable profile image should be allowed and the component handling the profile image should default to an icon. This repo is using grommet so check out this https://icons.grommet.io/?s=user `import { User } from 'grommet-icons';`. I think it would also be nice to use optional chaining better. I believe we can do this `spotifyUser?.images?.[0]?.url ?? null` https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

**Comment by adamdavies1915** [comment]
@Jonzy3000  please see the beautifully crafted code I've updated this PR with

File: src/components/common/Header.tsx [hunk 1]
```diff
@@ -20,7 +19,7 @@ export const Header = React.memo(() => {
         </Heading>
       </Button>
       {user.profileImage != null && (
-        <Avatar url={`url(${user.profileImage.url})`} />
+        <Avatar url={`url(${user.profileImage?.url ? user.profileImage.url : '/empty-avatar.png'})`} />
```
Comment (Jonzy3000) [inline src/components/common/Header.tsx]: Please use an icon here instead of the png, also this could be `user.profileImage?.url ?? '/empty-avatar.png'`

**Review by Jonzy3000 — CHANGES_REQUESTED** [review]

File: src/components/common/Header.tsx (hunk 1 above)
Comment (adamdavies1915) [reply src/components/common/Header.tsx]: How do you put an Icon inside a Avatar though?

Comment (Jonzy3000) [reply src/components/common/Header.tsx]: Might need a new component

**Pull request merged**